FieldworkAIDriver.lua

```diff
@@ -307,6 +307,13 @@ function FieldworkAIDriver:writeUpdateStream(streamId, connection, dirtyMask)
 		streamWriteIntN(streamId,self.convoyCurrentPosition,5)
 		streamWriteIntN(streamId,self.convoyTotalMembers,5)
 	end
+	if self.fieldworkState and self.fieldworkState ~= self.fieldworkStateSend then
```

Comment:
Sen**t**

Also wondering if it would make sense to create a wrapper function to set states (instead of directly setting them) and that could also take care of syncing it on MP so there is no need for this check